### ISSUE TITLE
Incorrect value(s) in REST API endpoint GET /api/projects/{project_id}

### ISSUE THREAD

**Issue by RothAndrew** [vmware/harbor#4537]
harbor v1.4

After pushing one repository to the project `library`:

`/api/projects` correctly returns "repo_count": 1

```
{
        "project_id": 1,
        "owner_id": 1,
        "name": "library",
        "creation_time": "2018-03-21T16:23:47Z",
        "update_time": "2018-03-21T16:23:47Z",
        "deleted": 0,
        "owner_name": "",
        "togglable": true,
        "current_user_role_id": 1,
        "repo_count": 1,
        "metadata": {
            "public": "true"
        }
    }
```

`/api/projects/1` incorrectly returns "repo_count": 0

```
{
    "project_id": 1,
    "owner_id": 1,
    "name": "library",
    "creation_time": "2018-03-21T16:23:47Z",
    "update_time": "2018-03-21T16:23:47Z",
    "deleted": 0,
    "owner_name": "",
    "togglable": false,
    "current_user_role_id": 0,
    "repo_count": 0,
    "metadata": {
        "public": "true"
    }
}
```

Looks like `current_user_role_id` and `togglable` are also affected.

![1](https://user-images.githubusercontent.com/16000938/38089524-b5e308b4-332d-11e8-8588-2a065a6791b4.PNG)
![2](https://user-images.githubusercontent.com/16000938/38089526-b5ff6d60-332d-11e8-971e-e8d5e77c7646.PNG)

**Comment by ywk253100** [comment]
Thanks @RothAndrew, the issue has been fixed by #4444 in master branch.

**Comment by RothAndrew** [comment]
Thanks. Any idea when the next release what includes this will be published?
